core/src/Kokkos_MemoryPool.hpp

```diff
@@ -74,6 +74,11 @@ class MemoryPool {
 
   enum : uint32_t { HINT_PER_BLOCK_SIZE = 2 };
 
+  static KOKKOS_FUNCTION unsigned integral_power_of_two_that_contains(
+      const unsigned N) {
+    return N ? Kokkos::bit_width(N - 1) : 0;
+  }
```

Comment:
Why are you reintroducing it here?

Reply:
It's more like I am preserving a named function for that utility since it is used 4x in `MemoryPool`.
It avoids repeating ourselves.